src/index.js

```diff
@@ -9,9 +9,47 @@ const ajv = new Ajv();
 
 Object.entries(schemas).forEach(([key, schema]) => ajv.addSchema(schema, key));
 
+const COMPARISON_REVERSION_MAP = {
+  endsWith: 'suffixOf',
+  equals: 'equals',
+  exists: 'exists',
+  in: 'includes',
+  includes: 'in',
+  notEquals: 'notEquals',
+  notIn: 'notIncludes',
+  notIncludes: 'notIn',
+  prefixOf: 'startsWith',
+  startsWith: 'prefixOf',
+  subset: 'superset',
+  suffixOf: 'endsWith',
+  superset: 'subset',
+};
+
+const USER_CUSTOM_ATTRIBUTES_PATH = 'user.customAttributes';
```

Comment:
@swain also, do we want to lock what values we replace? I assumed that we did and therefore only added support for `user.customAttributes`.

Reply:
I think the implementation here should not need to know about this path. We should be able to apply the reversing logic to any key which is known, not just this particular one.